.github/workflows/deployment-tests.yml

```
@@ -29,8 +29,7 @@ jobs:
       compose_file: docker-compose.dev.yaml
 
   deployment-prod-tests:
-    needs: changes-check
-    if: github.event_name == 'workflow_dispatch' || github.event.action == 'opened' || (github.event.action == 'synchronize' && needs.changes-check.outputs.any_changed == 'true')
+    needs: deployment-dev-tests
     uses: ./.github/workflows/deployment-test-template.yml
     with:
       compose_file: docker-compose.yaml
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)